content/posts/2021/reana-client-python-api.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "How to run workflows using reana-client Python API"
```

Comment:
How to run REANA workflows using Python API

(basically, here and elsewhere, we can loose a bit on the term `reana-client`, and just speak of REANA client etc, since people won't be seeing `reana-client` with dash much)